media/js/src/wizard.js

```diff
@@ -51,9 +88,17 @@
                 ul.append(pb);
             }
             if (index < (length - 1)) {
-                var nb = $('<li></li>').append($('<a>Next &gt;</a>')
-                    .attr({href:'#'}));
-                nb.click(function() {
+                var a = $('<a>Next &gt;</a>')
+                    .attr({href:'#', class:'btn next-button',
+                           id:'next-button-' + index});
+                if (index != 1) {
```

Comment:
Yes, I think we all know that '===' and '!==' are preferred in JS and why that is. When one is switching back and forth between JS and reasonable languages though, one's muscle memory tends to take over.